e2e-version/index.js

```diff
@@ -45,17 +48,31 @@ async function run() {
           if (semver.gt(pluginDependency, grafanaVersion.version)) {
             break;
           }
-          output.push(grafanaVersion.version);
+          versions.push(grafanaVersion.version);
         }
     }
 
     if (versionResolverType === VersionResolverTypes.PluginGrafanaDependency) {
-      // limit the number of versions to 6
-      output = evenlyPickVersions(output, VERSIONS_LIMIT);
+      // limit the number of versions to avoid starting too many jobs
+      versions = evenlyPickVersions(versions, VERSIONS_LIMIT);
     }
 
-    console.log('Resolved versions: ', output);
-    core.setOutput(MatrixOutput, JSON.stringify(output));
+    // official grafana images
+    const images = versions.map((version) => ({
+      name: 'grafana',
+      version,
+    }));
+
+    if (skipGrafanaDevImage !== 'true') {
```

Comment:
are you sure this variable doesn't come as boolean since you defined it `type: boolean`?

`false !== "true"` => true
`true !== "true" => true

Reply:
I think it did work. But after you raised this I did some more research, and I should definitely be using `core.getBooleanInput` instead of `core.getInput`.  Code updated. Thanks!